src/DropdownButton.jsx

```diff
@@ -62,10 +62,15 @@ var DropdownButton = React.createClass({
         'dropup': this.props.dropup
       };
```

Comment:
add `groupClasses[this.props.parentClassName] = true;` here instead of concatenating strings below.